components/admin_console/billing/billing_summary.tsx

```diff
@@ -45,9 +53,210 @@ const noBillingHistory = (
+                        />
+                        <OverlayTrigger
+                            delayShow={500}
+                            placement='bottom'
+                            overlay={(
+                                <Tooltip
+                                    id='BillingSubscriptions__seatOverageTooltip'
+                                    className='BillingSubscriptions__tooltip BillingSubscriptions__tooltip-right'
+                                    positionLeft={390}
+                                >
+                                    <div className='BillingSubscriptions__tooltipTitle'>
+                                        <FormattedMessage
+                                            id='admin.billing.subscriptions.billing_summary.lastInvoice.whatArePartialCharges'
+                                            defaultMessage='What are partial charges?'
+                                        />
+                                    </div>
+                                    <div className='BillingSubscriptions__tooltipMessage'>
+                                        <FormattedMessage
+                                            id='admin.billing.subscriptions.billing_summary.lastInvoice.whatArePartialCharges.message'
+                                            defaultMessage='Users who have not been enabled for the full duration of the month are charged at a prorated monthly rate.'
+                                        />
+                                    </div>
+                                </Tooltip>
+                            )}
+                        >
+                            <i className='icon-information-outline'/>
+                        </OverlayTrigger>
+                    </div>
+                }
+                {partialCharges.map((charge) => (
+                    <div
+                        key={charge.price_id}
+                        className='BillingSummary__lastInvoice-charge'
+                    >
+                        <div className='BillingSummary__lastInvoice-chargeDescription'>
+                            <FormattedMarkdownMessage
+                                id='admin.billing.subscriptions.billing_summary.lastInvoice.userCountPartial'
+                                defaultMessage='{users} users'
+                                values={{users: charge.quantity}}
+                            />
+                        </div>
+                        <div className='BillingSummary__lastInvoice-chargeAmount'>
+                            <FormattedNumber
+                                value={(charge.total / 100.0)}
+                                // eslint-disable-next-line react/style-prop-object
+                                style='currency'
+                                currency='USD'
+                            />
+                        </div>
+                    </div>
+                ))}
+                {invoice.tax &&
```

Comment:
If there are no taxes, I believe we should still show the line but with 0 as the value to be consistent with how it is presented in the actual invoice

Reply:
I feel like it's going to waste screen real estate for many users, especially in the beginning where apparently we're only charging tax for a subset of customers. I don't think people need to see that they have a 0 charge for something that's not relevant to them.

Reply:
@matthewbirtch any thoughts on this whether to completely hide the Tax line if it's zero?

Reply:
I think it's fine to hide the tax line if it's zero. This is just a summary view anyway, so it's reasonable that we don't show lines that are irrelevant. My 2cents, but not sure if it's super important from a finance perspective